### DISCUSSION TITLE
Authelia is constantly redirecting to the login page when using Swag as a reverse proxy

### DISCUSSION THREAD

**Discussion by steamch** [authelia/authelia#5045] [Q&A] [answered]
hi,there
When I use Swag as a reverse proxy for Authelia, after logging in, the page will redirect back to the login page

`
###############################################################################
#                           Authelia Configuration                            #
###############################################################################

theme: dark
jwt_secret: "YOUR_JWT_SECRET"
default_redirection_url: https://authelia.xxxx.xxxx.com:183/

server:
  host: 0.0.0.0
  port: 9091
  path: "authelia"
  enable_pprof: false
  enable_expvars: false
  disable_healthcheck: false
  tls:
    key: ""
    certificate: ""

log:
  level: info

totp:
  issuer: YOUR_DOMAIN.COM
  period: 30
  skew: 1

authentication_backend:
  password_reset:
    disable: false
  refresh_interval: 5m
  file:
    path: /config/users_database.yml
    password:
      algorithm: argon2id
      iterations: 1
      key_length: 32
      salt_length: 16
      memory: 1024
      parallelism: 8

access_control:
  default_policy: deny
  rules:
    ## bypass rule
    - domain: 
        - "authelia.xxxx.xxxx.com:183"
      policy: one_factor

session:
  name: authelia_session
  domain: authelia.xxxx.xxxx.com:183
  same_site: lax
  secret: BMWNCRqvAtgC1llrm63xxxxE9O5WVKUcIhyhAykaptKAGlxZ85tRD
  expiration: 1h
  inactivity: 5m
  remember_me_duration: 2M
  redis:
    host: redis
    port: 6379
    password: "xxxx123"
    database_index: 0
    maximum_active_connections: 10
    minimum_idle_connections: 0

regulation:
  max_retries: 3
  find_time: 10m
  ban_time: 12h

storage:
  encryption_key: zRE7wE3RLbTRVxeEfrvanw0LDSxxxxZIukhtTnrccu86u
  mysql:
    host: mariadb
    port: 3306
    database: authelia
    username: authelia
    password: "xxxx123"
  
notifier:
  disable_startup_check: false
  smtp:
    username: xxxx@outlook.com
    password: "xxxx"
    host: smtp.office365.com
    port: 587
    sender: ojifood@outlook.com
    identifier: localhost
    subject: "[Authelia] {title}"
    startup_check_address: test@authelia.com
    disable_require_tls: false
    disable_starttls: false
    disable_html_emails: false
    tls:
      skip_verify: false
      minimum_version: TLS1.2
`
please help me thank you

**Comment by james-d-elliott** [comment, marked as answer]
Remove :183 from the session domain. You'll need to provide logs after doing this if it isn't the solution.

**Reply by steamch** [reply]
yes,it fixed.thank you
